lang/python/python3-drf-nested-routers/Makefile

```diff
@@ -1,17 +1,18 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=drf-nested-routers
-PKG_VERSION:=0.91
-PKG_RELEASE:=3
+PKG_VERSION:=0.92.1
+PKG_RELEASE:=1
 
 PYPI_NAME:=drf-nested-routers
-PKG_HASH:=46e5c3abc15c782cafafd7d75028e8f9121bbc6228e3599bbb48a3daa4585034
+PKG_HASH:=e043fc937f94ac462a92d2d9fc9a7e55710a67164b558442adfe9634fc519c3b
 
 PKG_LICENSE:=Apache-2.0
 PKG_LICENSE_FILES:=LICENSE
 PKG_MAINTAINER:=Peter Stadler <peter.stadler@student.uibk.ac.at>
 
 include ../pypi.mk
+
```

Comment:
why?

Reply:
I forgot to remove this after going back to the pypi version, corrected it now. Thank you :-)